cmake/rocksdb_flags.cmake

```diff
@@ -6,42 +6,52 @@ option(WITH_JEMALLOC "build with JeMalloc" OFF)
 if (WITH_JEMALLOC)
   find_package(JeMalloc REQUIRED)
   add_definitions(-DROCKSDB_JEMALLOC -DJEMALLOC_NO_DEMANGLE)
-  include_directories(${JEMALLOC_INCLUDE_DIR})
+  include_directories(${JeMalloc_INCLUDE_DIRS})
```

Comment:
why have these changes

Reply:
We use `rocksdb/cmake/modules/*` to resolve these libraries. And the names in rocksdb is changed.